### PR TITLE
Updated MANIFEST and add autover versioning

### PR DESCRIPTION
**Pull request by jlstevens**
Updates the MANIFEST to include the license file and uses param both to add an ``id`` parameter to ``Comms`` and for versioning with autover.

### CODE REVIEW

**Comment by jlstevens** [comment]
@philippjfr I think this is ready to review/merge now. Once it is on master I'll update the tags.

File: setup.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def get_setup_version(reponame):
     license='BSD',
     url='http://pyviz.org',
     packages = ["pyviz_comms"],
+    package_data={'pyviz_comms': ['.version']},
```
Comment (philippjfr) [inline setup.py]: Shouldn't it be sufficient to add it to the MANIFEST?

Comment (jlstevens) [reply setup.py]: No! I checked multiple times. Without it, the ``.version`` file doesn't end up in the archive built with ``conda build .``. I don't get it either but it is needed...

Comment (philippjfr) [reply setup.py]: Okay, just noticed the other packages using autover also have it.

Comment (ceball) [reply setup.py]: @jlstevens/@philippjfr, I made a note about this:
https://github.com/pyviz/pyct/wiki/Packaging-tips

E.g. https://github.com/pyviz/nbsite/blob/master/setup.py

Note you need to be worried not only what ends up in a package, but also what gets installed.

It'll be great when we do it uniformly, but right now we're using package_data in many projects (for the examples, usually).

**Pull request merged**